From 9a070d8d236e1961ac05ae71e39d0bc531e0c3d4 Mon Sep 17 00:00:00 2001
From: Lawrence Mok <lawrence.mok@amlogic.com>
Date: Wed, 30 Jul 2014 16:32:58 -0700
Subject: [PATCH 4605/5965] remove "platform" part of /proc/cpuinfo Hardware
 string

/proc/cpuinfo used to show
    Hardware        : Amlogic Meson8B platform
with this change:
    Hardware        : Amlogic Meson8B

It's unnecessary and fixes Android hardware specific init.rc loading.

Change-Id: I898cc09e0f43718ddaceae9c6fd4a925f6bf4da1
---
 arch/arm/mach-meson6/board-meson6-common.c       | 2 +-
 arch/arm/mach-meson6tv/board-meson6tv-common.c   | 2 +-
 arch/arm/mach-meson6tvd/board-meson6tvd-common.c | 2 +-
 arch/arm/mach-meson8/board-meson8-common.c       | 2 +-
 arch/arm/mach-meson8b/board-meson8b-common.c     | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/mach-meson6/board-meson6-common.c b/arch/arm/mach-meson6/board-meson6-common.c
index 2b4163ebebf3..7ded9506a750 100644
--- a/arch/arm/mach-meson6/board-meson6-common.c
+++ b/arch/arm/mach-meson6/board-meson6-common.c
@@ -174,7 +174,7 @@ static const char *m6_common_board_compat[] __initdata = {
 	NULL,
 };
 
-DT_MACHINE_START(AML8726_MX, "Amlogic Meson6 platform")
+DT_MACHINE_START(AML8726_MX, "Amlogic Meson6")
 	.reserve	= meson6_reserve,
 //.nr_irqs	= 
 	.smp		= smp_ops(meson_smp_ops),
diff --git a/arch/arm/mach-meson6tv/board-meson6tv-common.c b/arch/arm/mach-meson6tv/board-meson6tv-common.c
index 5f73886c56d6..9c8707c304ee 100755
--- a/arch/arm/mach-meson6tv/board-meson6tv-common.c
+++ b/arch/arm/mach-meson6tv/board-meson6tv-common.c
@@ -112,7 +112,7 @@ static const char __initdata *m6tv_common_boards_compat[] = {
 	NULL,
 };
 
-DT_MACHINE_START(AML8726_MX, "Amlogic Meson6TV platform")
+DT_MACHINE_START(AML8726_MX, "Amlogic Meson6TV")
 	.smp		= smp_ops(meson6tv_smp_ops),
 	.map_io		= meson6tv_map_io,	// dt - 1
 	.init_early	= meson6tv_init_early,	// dt - 2
diff --git a/arch/arm/mach-meson6tvd/board-meson6tvd-common.c b/arch/arm/mach-meson6tvd/board-meson6tvd-common.c
index 9ff57946a497..45d1b9b86f3e 100755
--- a/arch/arm/mach-meson6tvd/board-meson6tvd-common.c
+++ b/arch/arm/mach-meson6tvd/board-meson6tvd-common.c
@@ -112,7 +112,7 @@ static const char __initdata *m6tv_common_boards_compat[] = {
 	NULL,
 };
 
-DT_MACHINE_START(AML8726_MX, "Amlogic Meson6TVD platform")
+DT_MACHINE_START(AML8726_MX, "Amlogic Meson6TVD")
 	.smp		= smp_ops(meson6tv_smp_ops),
 	.map_io		= meson6tv_map_io,	// dt - 1
 	.init_early	= meson6tv_init_early,	// dt - 2
diff --git a/arch/arm/mach-meson8/board-meson8-common.c b/arch/arm/mach-meson8/board-meson8-common.c
index 4eb79966a1a9..693bb7478b6f 100755
--- a/arch/arm/mach-meson8/board-meson8-common.c
+++ b/arch/arm/mach-meson8/board-meson8-common.c
@@ -142,7 +142,7 @@ static const char *m8_common_board_compat[] __initdata = {
 	NULL,
 };
 
-DT_MACHINE_START(AML8726_M8, "Amlogic Meson8 platform")
+DT_MACHINE_START(AML8726_M8, "Amlogic Meson8")
 	.reserve	= meson8_reserve,
 //.nr_irqs	= 
 	.smp		= smp_ops(meson_smp_ops),
diff --git a/arch/arm/mach-meson8b/board-meson8b-common.c b/arch/arm/mach-meson8b/board-meson8b-common.c
index 6a95585f63f5..dce50a8e0f4e 100644
--- a/arch/arm/mach-meson8b/board-meson8b-common.c
+++ b/arch/arm/mach-meson8b/board-meson8b-common.c
@@ -132,7 +132,7 @@ static const char *m8_common_board_compat[] __initdata = {
 	NULL,
 };
 
-DT_MACHINE_START(AML8726_M8, "Amlogic Meson8B platform")
+DT_MACHINE_START(AML8726_M8, "Amlogic Meson8B")
 	.reserve	= meson8b_reserve,
 //.nr_irqs	= 
 	.smp		= smp_ops(meson_smp_ops),
-- 
2.19.0

